core/src/block.rs

```diff
@@ -659,6 +493,269 @@ impl From<&ValidBlock> for Vec<Event> {
     }
 }
 
+declare_versioned_with_scale!(VersionedCandidateBlock 1..2, Debug, Clone, iroha_macro::FromVariant, IntoSchema);
+
+impl VersionedCandidateBlock {
+    /// Converts from `&VersionedCandidateBlock` to V1 reference
```

Comment:
```suggestion
    /// Convert from `&VersionedCandidateBlock` to V1 reference
```

the verbs should be in imperative (without `s`)

applies to other descriptions as well